conformance/report_template.gohtml

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+<style>
+    table, th, td {
+        border: 1px solid black;
+    }
+    td {
+        padding: 2px;
+    }
+</style>
+<body>
+<h2>MCS Conformance Report</h2>
+
+{{range .Groups}}
+<h3>{{.Name}}</h3>
+<table>
+    <th>Result</th>
+    <th>Description</th>
+    <th>Reference</th>
+    {{range .Tests}}
+    <tr>
+        {{ if .Pass }}
+        <td style="color:green">Pass</td>
+        {{ else }}
+        <td style="color:red">Fail</td>
+        {{end}}
+        <td>{{.Desc}}</td>
+        <td><a href="{{.Ref}}">{{.Ref}}</a></td>
+    </tr>
```

Comment:
I don’t think it’s all that useful to have the link appear as text; let’s attach it to the description:

```suggestion
    <tr>
        <th>Result</th>
        <th>Description</th>
    </tr>
    {{range .Tests}}
    <tr>
        {{ if .Pass }}
        <td style="color:green">Pass</td>
        {{ else }}
        <td style="color:red">Fail</td>
        {{end}}
        <td><a href="{{.Ref}}">{{.Desc}}</a></td>
    </tr>
```